package.json

```diff
@@ -38,8 +38,9 @@
     "react": "^16.8.0"
   },
   "dependencies": {
-    "@wayke-se/ecom": "^2.2.0",
-    "marked": "^0.8.1"
+    "@wayke-se/ecom": "2.3.0-alpha.3",
+    "marked": "^0.8.1",
+    "react-dom": "^16.13.1"
```

Comment:
Why do we need `react-dom` as a dependency? :thinking:

Reply:
Can't remember adding it, I'll see if it is possible to remove it.